e2e/common/misc/pipe_test.go

```diff
@@ -38,6 +40,12 @@ import (
 func TestPipe(t *testing.T) {
 	t.Parallel()
 
+	// [TODO] Enable arm64 smoke tests to run out of the box
+	// https://github.com/apache/camel-k/issues/5362
+	if runtime.GOARCH == "arm64" && os.Getenv("CAMEL_K_TEST_BASE_IMAGE") == "" {
+		t.Skipf("TestPipe not supported on platform: %s/%s", runtime.GOOS, runtime.GOARCH)
```

Comment:
Why is this test not supported? I don't think it should fail whichever the platform running it is.